pom.xml

```
@@ -17,7 +17,7 @@
         <failsafe-plugin.version>3.2.5</failsafe-plugin.version>
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-        <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
+        <quarkus.platform.version>3.7.2</quarkus.platform.version>
         <quarkus.ide.version>3.7.0</quarkus.ide.version>
```

can you synchronize it,please?
```suggestion
        <quarkus.ide.version>3.7.2</quarkus.ide.version>
```

---

Done